docker-compose.test.yml

```diff
@@ -4,7 +4,7 @@ services:
   test:
     image: cypress/included:7.3.0
     entrypoint: []
-    command: /bin/bash -c "npx wait-on http://ckan:5000 && cypress run"
+    command: /bin/bash -c "npx wait-on http://ckan:5000/dataset && cypress run"
```

Comment:
This shouldn't matter.  The test work locally for me without this change.  I think this might just be another blip in catalog tests unfortunately 😕

Reply:
Bringing it up locally, one of the requirements bumps (jinja2 dependencies being loose) causes the app to fail to load. @robert-bryson will push a fixed version... We thought maybe npx didn't like the 302 that occurs when hitting root and that hitting /dataset would work cleaner, but it was the requirement bump that was the issue.

Reply:
Interesting haha.. I didn't rebuild, so yeah, that makes sense 😅